src/screens/HomePage/HomePage.tsx

```diff
@@ -19,12 +18,7 @@ import HomepageStructuredData from 'screens/HomePage/HomepageStructuredData';
 import { filterGeolocatedRegions } from 'common/regions';
 import { useGeolocatedRegions, useShowPastPosition } from 'common/hooks';
 import HomePageHeader from 'components/Header/HomePageHeader';
-import {
-  Content,
-  SectionWrapper,
-  Section,
-  ColumnCentered,
-} from './HomePage.style';
+import { Content, Section, ColumnCentered } from './HomePage.style';
```

Comment:
we can now delete the `SectionWrapper` styled component from `HomePage.style.tsx`